lightningd/options.c

```diff
@@ -528,6 +548,8 @@ static const struct config testnet_config = {
 	.max_fee_multiplier = 10,
 
 	.use_dns = true,
+
+	.dust_limit = { 546 },
```

Comment:
Just noticed, you missed to add the `dust_limit`  value to the wallet teststub config in `wallet/test/test_utils.c`.
It compiles without it, but Valgrind will be unhappy when a DB migration will be using this field in the future...

Reply:
@niftynei    See? It will be missed   ;)